docs/game_sdk/Users.md

```diff
@@ -142,6 +142,21 @@ Fires when the `User` struct of the currently connected user changes. They may h
 | ---- | ---- | -------------------------------------- |
 | user | User | a new User struct for the current user |
 
+###### Example
+
+```cs
+var userManager = discord.GetUserManager();
+// GetCurrentUser will error until this fires once.
+userManager.OnCurrentUserUpdate += () =>
+{
+var currentUser = userManager.GetCurrentUser();
+Console.WriteLine(currentUser.Username);
+Console.WriteLine(currentUser.Id);
+Console.WriteLine(currentUser.Discriminator);
+Console.WriteLine(currentUser.Avatar);
```

Comment:
Some spacing issues here; this should be formatted like C# (4 spaces I think?)

Reply:
I did notice that as well, but was unsure whether to keep it as such or not... Since I just grabbed it from Program.cs .

Reply:
Spaced out! Respectively.